src/com/tutorial/main/GameObject.java

```diff
@@ -1,55 +0,0 @@
-package com.tutorial.main;
-
-import java.awt.Graphics;
-import java.awt.Rectangle;
-
-public abstract class GameObject {
```

Comment:
I think it means GameObject is a set of variables and methods that all subclasses share, but needs a subclass to be instantiated. So I couldn't do 

`new GameObject`

right? Would GameObject be considered a superclass for subclasses?